src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -76,13 +76,13 @@ def phylo_trees():
             .options(joinedload(phylo_run_alias.outputs))
             .filter(
                 or_(
+                    user.system_admin,
                     phylo_run_alias.group_id.in_(
                         db_session.query(Group.id)
                         .join(User)
                         .filter(User.auth0_user_id == profile["user_id"])
                         .subquery()
                     ),
-                    user.system_admin,
```

Comment:
wait so moving this up is what fixed it? That seems really weird to me

Reply:
In this case, I just wanted to group the `phylo_run_alias.group_id.in_(` statements.